torch/csrc/api/include/torch/expanding_array.h

```diff
@@ -26,7 +26,13 @@ class ExpandingArray {
   /*implicit*/ ExpandingArray(std::initializer_list<T> list)
       : ExpandingArray(at::ArrayRef<T>(list)) {}
 
-  /// Constructs an `ExpandingArray` from an `initializer_list`. The extent of
+  /// Constructs an `ExpandingArray` from an `std::vector`. The extent of
+  /// the length is checked against the `ExpandingArray`'s extent parameter `D`
+  /// at runtime.
+  /*implicit*/ ExpandingArray(std::vector<T> vec)
```

Comment:
wait -- this doesn't match the actual C++ API?  (torch::tensor).  That doesn't seem good.